developer-docs-site/docs/releases/devnet-release.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Devnet Release"
+slug: "devnet-release"
+---
+
+# Latest Devnet Release
+
+The table below contains latest release information for each Aptos component in the mainnet network. We also suggest you monitor the [#devnet-release](https://discord.com/channels/945856774056083548/956692649430093904) channel of [Aptos Discord](https://discord.gg/aptoslabs) for ongoing updates.
+
+See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub notifications and more release-related information.
+
+## Aptos CLI
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
+|---|---|---|---|---|
+|[Aptos CLI Release v1.0.7](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-cli-v1.0.7)| `aptos-cli-v1.0.7` | `?` | ?| `?` |
+
+## Aptos Framework
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
```

Comment:
I don't think docker image tag is relevant here, instead maybe a link to the docs?

Also we should keep track here of which version is on which network

Reply:
I have removed the *Docker Image Tag* column from the Aptos Framework table on all three network release pages.

@perryjrandall , please help me understand what you mean by which network. Note we have a release page dedicated to each here.